code/obj/decorations.dm

```diff
@@ -175,11 +180,45 @@
 				qdel(src)
 			else
 				src.take_damage(45)
+
 	attack_hand(mob/user)
 		if (!user) return
-		if (destroyed) return ..()
+		if (destroyed && iscow(user) && user.a_intent == INTENT_HELP)
+			boutput(user, "<span class='notice'>You pick at the ruined bush, looking for any leafs to graze on, but cannot find any.</span>")
+			return ..()
+		else if (destroyed)
+			return ..()
 
 		user.lastattacked = src
+		if (iscow(user) && user.a_intent == INTENT_HELP)	//Cow people may want to eat some of the bush's leaves
+			src.bites -= 1
+			var/desired_mask = (src.bites / initial(src.bites)) * 5
+			desired_mask = round(desired_mask)
+			desired_mask = max(1,desired_mask)
+			desired_mask = min(desired_mask, 5)
+
+			if (desired_mask != current_mask)
+				current_mask = desired_mask
+				src.add_filter("bite", 0, alpha_mask_filter(icon=icon('icons/obj/foodNdrink/food.dmi', "eating[desired_mask]")))
+
+			eat_twitch(user)
+			playsound(user, "sound/items/eatfood.ogg", rand(10,50), 1)
+
+			if (is_plastic)
+				user.setStatus("weakened", 3 SECONDS)
+				user.visible_message("<span class='notice'>[user] takes a bite out of [src] and chokes on the plastic leaves.</span>", "<span class='alert'>You munch on some of [src]'s leaves, but realise too late it's made of plastic. You start choking!</span>")
+				user.take_oxygen_deprivation(20)
+				user.losebreath+=2
+			else
+				user.changeStatus("food_hp_up", 20 SECONDS)
+				user.visible_message("<span class='notice'>[user] takes a bite out of [src].</span>", "<span class='notice'>You munch on some of [src]'s leaves, like any normal human would.</span>")
+				var/mob/living/carbon/human/H = user
+				H.sims?.affectMotive("Hunger", 10)
+
+			if(src.bites <= 0)
+				destroy()
+			return 0
```

Comment:
this should be in its own proc and then you override that in the child

Reply:
Done, thanks for the heads up!